src/FakeItEasy/FakeItEasy.nuspec

```diff
@@ -13,10 +13,41 @@
     <licenseUrl>https://github.com/FakeItEasy/FakeItEasy/blob/master/License.txt</licenseUrl>
     <copyright>Copyright (c) FakeItEasy contributors. (fakeiteasyfx@gmail.com)</copyright>
     <tags>TDD unittesting mocks mocking fakes faking stubs stubbing spy spies doubles isolation substitutes substitution</tags>
+    <dependencies>
+      <group targetFramework=".NETFramework4.0">
+        <dependency id="Newtonsoft.Json" version="9.0.1" />

```

Comment:
Personally I'd rather disable the feature in .NET Core than use JSON.NET instead of binary serialization.